herddb-net/src/main/java/herddb/network/netty/NettyChannelAcceptor.java

```diff
@@ -249,7 +249,7 @@ public Integer getSample() {
         });
         InetSocketAddress address = new InetSocketAddress(host, port);
         LOGGER.log(Level.INFO, "Starting HerdDB network server at {0}:{1}", new Object[]{host, port + ""});
-        if (address.isUnresolved()) {
+        if (enableRealNetwork && address.isUnresolved()) {

```

Comment:
Is this related to multiple local databases?

Reply:
yes, because we are going to use the name of the database as "hostname".
jdbc:herddb:local:mydb

as we are using Netty Local Channel we won't need this to be a valid hostname.
we need it to be a resolved hostname when we are using real network and especially when you are using Kerberos based auth

so in case of "local" we can skip that check and we must do it otherwise it will fail